src/multiplayer-servers/multiplayer-services/server-protection/gateway-policies.md

```diff
@@ -0,0 +1,50 @@
+# Gateway Policies
+
+Gateway policies provide an additional layer of protection for your game infrastructure by 
+redirecting specific outgoing game server traffic to destinations such as your game backend.
+
+If SteelShield is active while unaware of your legitimate outgoing traffic, it might block connection attempts or responses that are not meant to be filtered.
+
+## Attack Scenario
+
+Without gateway policies, an attacker can spoof a game host's IP address and send malicious requests to your game backend. 
+In response, you or your hosting provider might decide to block the IP address. 
+As a result, your game host, including legitimate game server traffic, 
+would no longer be able to communicate with your game backend.
+
+By redirecting traffic from your game server through the gateway, 
+the traffic originates from gateway IP addresses that are typically unknown to attackers. 
+This makes spoofing a game host's IP address [largely](#exceptions) ineffective.
```

Comment:
Add

> ineffective at disrupting communications with your backend server

since the spoofing could still be effective at disrupting the game client to game server communication (large scale reflection attack that overwhelms the uplink capacity)